blog/2020-10-12-ambrosia-alpha-release.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Trade Bitcoin and DAI with COMIT"
+author: franck
+author_url: https://github.com/d4nte
+author_image_url: https://avatars1.githubusercontent.com/u/300805
+tags: [bitcoin,dai,atomic swap,ambrosia]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# A Change of Direction
+
+Last year, we spent time and effort creating tools, such as [`create-comit-app` and `comit-scripts`](https://github.com/comit-network/create-comit-app/) to allow developers to build their own COMIT Apps.
+We organised local meetups in Sydney and even a hackathon in Singapore.
+We had a number of developers excited about the technology and possibilities that COMIT bring in but very few committed to build an app.
+
+We know realize that before other build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
+
+# Ambrosia
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-10-12/ambrosia.png')} />
+
+This is why we spent the last few months working on [Ambrosia](https://github.com/comit-network/ambrosia/), an UI for trading in the COMIT network.
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) has a bakend.
+
+The software has some rough edges, but we wanted to get it out there to have your feedback. Here are some choices we want your input on:
+
+## Custody
+
+Not only you do not need to trust a third party with your fund when trading, even our software is non-custodial.
+Ambrosia never holds your crypto, it interfaces with your hardware wallet (nano ledger s only) to make payments.
+
+We believe that custody and security are important criteria to our users, hence we also assume that users are running their own [Bitcoin Core](https://github.com/bitcoin/bitcoin) node.
+
+## Trading pair
+
+The COMIT Network Daemon (cnd) supports various trading pairs: BTC/ETH, BTC/ERC20, Lightning BTC/ETH, etc.
+However, we have limited Ambrosia to the BTC/DAI pair as we know that Bitcoin/USD.. are usually the most popular pair.
+
+## Decentralised Orderbook
+
+At this stage, Ambrosia can only take offers.
+We have a [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
```

Comment:
Do we want to mention that? :D

Reply:
yeah in case one try to swap with another ambrosia

Reply:
Well, what about the "don't mention publicly that we are running a nectar instance"? I think @bonomat had concerns here :man_shrugging:

Reply:
Good point. Please change it to show that this is only for test purposes :) 
```suggestion
For test purposes only we have a temporary [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
```